src/Dapr.Client/GetWorkflowResponse.cs

```diff
@@ -19,15 +19,15 @@ namespace Dapr.Client
     /// <summary>
     /// Represents the response from invoking a binding.
     /// </summary>
-    public sealed class GetWorkflowResponse
+    public sealed record GetWorkflowResponse
```

Comment:
This actually isn't the right way to use `record`, so I'm a little surprised that C# lets you do this. The right way would be to do something like this:

```csharp
public record GetWorkflowResponse(string InstanceId, DateTime StartTime, IReadOnlyDictionary<string, string> Metadata);
```

And that's it - you can remove your constructor and the properties below because the C# compiler will auto-generate them.